src/components/PupilComponents/QuizEngineProvider/QuizEngineProvider.test.tsx

```diff
@@ -302,33 +228,17 @@ describe("QuizEngineContext", () => {
     });
 
     it("should grade a multi answer mcq as incorrect if only some of the pupilAnswers are correct", () => {
-      if (!questionsArrayFixture) {
-        throw new Error("questionsArrayFixture is null");
-      }
-
-      const multiQs = [...questionsArrayFixture].filter(
-        (question) => question.questionType === "multiple-choice",
+      const multiQs = structuredClone(
+        questionsArrayFixture.filter(
+          (question) => question.questionType === "multiple-choice",
+        ),
       );
-
-      // set the first answer as also correct
-      if (multiQs[0]?.answers?.["multiple-choice"]?.[0]) {
-        multiQs[0].answers["multiple-choice"][0].answer_is_correct = true;
```

Comment:
This was mutating a global fixture which could result in unexpected and hard to trace test failures down the road.